src/TropicalGeometry/linear_space.jl

```diff
@@ -126,7 +126,7 @@ end
 @doc raw"""
     tropical_linear_space(k::Int, n::Int, p::Vector{<:TropicalSemiringElem}; weighted_polyhedral_complex_only::Bool=false)
 
-Return a tropical linear space from a tropical Pluecker vector with indices `AbstractAlgebra.combinations(1:n,k)` and values `p`.  If `weighted_polyhedral_complex==true`, will not cache any extra information.
+Return a tropical linear space from a tropical Pluecker vector with indices `combinations(n,k)` and values `p`.  If `weighted_polyhedral_complex==true`, will not cache any extra information.
```

Comment:
```suggestion
Return a tropical linear space from a tropical Pluecker vector with indices `combinations(n,k)` and values `p`.  If `weighted_polyhedral_complex_only==true`, will not cache any extra information.
```
not your fault, but the kwarg in the signature is called slightly different